docs/ql-documentation/learn-ql/python/pointsto-type-infer.rst

```diff
@@ -186,38 +191,38 @@ The original query looked this:
    import python
 
    from Call call, Name name
-   where call.getFunc() = name and name.getId() = "input"
-   select call, "call to 'input'."
+   where call.getFunc() = name and name.getId() = "eval"
+   select call, "call to 'eval'."
 
-➤ `See this in the query console <https://lgtm.com/query/690010037/>`__. Two of the demo projects on LGTM.com have calls that match this pattern.
+➤ `See this in the query console <https://lgtm.com/query/6718356557331218618/>`__. Two of the demo projects on LGTM.com have calls that match this pattern.
```

Comment:
Since that link has three projects with results (1 non-demo project), suggest changing to "Some of the...".